nilearn/glm/tests/test_second_level.py

```diff
@@ -1345,6 +1346,7 @@ def test_second_level_t_contrast_length_errors(n_subjects):
         model.compute_contrast(second_level_contrast=[1, 2])
 
 
+@pytest.mark.timeout(300)
```

Comment:
```suggestion
@pytest.mark.slow
```

let's mark the tests as slow so they are run in separately with no timeout